tools/installers/install_muskit.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if [ $# != 0 ]; then
+    echo "Usage: $0"
+    exit 1;
+fi
+
+rm -rf muskit.done
+rm -rf ParallelWaveGAN
+git clone https://github.com/kan-bayashi/ParallelWaveGAN.git
+cd ParallelWaveGAN
```

Comment:
Is ParallelWaveGAN the only file to be loaded? I thought we also have some other packages (e.g., miditoolkit)